### PR TITLE
i18n for public website and documentation and French translation of existing doc

### PR DESCRIPTION
**Pull request by mastogros**
tl;dr : Add i18n capabilities with a language switcher, modify site structure by adding "en" and "fr" repositories, translate every existing page in French.
-

I've added 2 directories : en and fr.
I've put all the existing english doc in the "en" directory.
Then copied everything to the "fr" directory and translated every page.

I've added a "language switcher" which is basically a select list that will check the languages.js file and create the options for the select.
Note that the select will only appear if the same page exists in other languages.
To know if a page exists in other language, I've added a new attribute : "translationKey" in the pages files.
For example, in the homepage it's "translationKey : homepage", in the Usage with Nextcloud page it's "translationKey: usage-nextcloud"
The "fr", "en" (and then other languages) files that corresponds to the translation must contain the same translationKey.

I can add more documentation, but first I've translated the existant and if it is OK with this structure, I can help to add more doc.

PS : the more I use photoview the more I love it ;)

### CODE REVIEW

**Comment by viktorstrate** [comment]
Nice work! My only concern is that currently a lot of old links would be broken. Would it be possible to have all urls not prefixed with a language code automatically be redirected to the english one?

For example `/docs` -> `/en/docs`, instead of 404

**Comment by mastogros** [comment]
@viktorstrate you're right 👍  I'm going to check how this can be done

**Comment by mastogros** [comment]
@viktorstrate it seems there's no easy way to have redirections in Github pages, so I chose the best option : meta refresh+ canonical link, this way search engines will interpret as 301 redirections. If you choose to migrate from github pages to another thing, this can be replaces by a .htaccess file or a _redirects with Netlify for example.
And for end-users it works well.

**Comment by viktorstrate** [comment]
That's good for now, it won't be hard to change in the future, if we decide to move to somthing other than GH Pages.
Thank you very much for your work and effort!
